src/ota_proxy/external_cache.py

```diff
@@ -25,12 +25,23 @@
 
 
 def mount_external_cache(
-    mnt_point: StrOrPath, *, cache_dev_fslabel: str = config.EXTERNAL_CACHE_DEV_FSLABEL
+    mnt_point: StrOrPath,
+    is_nfs_cache: bool = False,
+    *,
```

Comment:
It would be better to set `is_nfs_cache` as a keyword argument.
```suggestion
    *,
    is_nfs_cache: bool = False,
```